src/universe/party.cpp

```diff
@@ -792,9 +792,12 @@ void cParty::writeTo(std::ostream& file) const {
 		}
 	file << '\f';
 	file << '\f';
-	for(unsigned int i = 0; i < party_event_timers.size(); i++)
+	for(unsigned int i = 0; i < party_event_timers.size(); i++) {
+		if (party_event_timers[i].node<=-200 || (party_event_timers[i].node>=-9 && party_event_timers[i].node<0))
```

Comment:
Can you explain this condition? Why are both not simply `<= 0`? Is there a reason `time` is not checked?

Reply:
Yes, just check that party_event_timers.node<0 : I guess I was trying to remove the export of unnecessary timers, and wasn't sure if magic_ptrs and pointers were used when party_event_timers.node<0 :-~

**Note**: Of course, you can also check party_event_timers.node.time

Reply:
Looks like pointers are not used in this context, so I'll just check `< 0` then. Node 0 should also be valid, after all.